.dockerignore

```diff
@@ -0,0 +1,29 @@
+# We cannot call `docker build` out of top dir since this is controlled by
```

Comment:
Wasn't this added in a different PR?

Reply:
Not at this project, but I can remove it from here.

Reply:
Not at this project, but I can remove it from here.